dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm

```diff
@@ -410,65 +410,6 @@ sub process_file {
     or $self->death("Failed to parse XS file\n");
   $AST->as_code($self);
 
-  # ----------------------------------------------------------------
```

Comment:
"ParseXS: refactor: add Node::pre_boot " commit message typo "and any user XSUBs
**emiited**"